### PR TITLE
chore: update componentCls key again

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: https://ant.design/changelog

正则全局匹配：``\[`\$\{\s*([^\}]*)\s*\}`\]``

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | chore: update componentCls key again |
| 🇨🇳 Chinese | chore: update componentCls key again |

### CODE REVIEW

File: components/list/style/index.ts [hunk 1]
```diff
@@ -127,7 +127,7 @@ const genResponsiveStyle = (token: ListToken): CSSObject => {
     },
 
     [`@media screen and (max-width: ${screenSM}px)`]: {
-      [`${componentCls}`]: {
+      [componentCls]: {
```
Comment (afc163) [inline components/list/style/index.ts:130]: 有没有 biome lint 的规则能防止再写这个。

File: components/skeleton/style/index.ts [hunk 2]
```diff
@@ -254,7 +254,7 @@ const genBaseStyle: GenerateStyle<SkeletonToken> = (token: SkeletonToken) => {
   } = token;
 
   return {
-    [`${componentCls}`]: {
+    [skeletonButtonCls]: {
```
Comment (afc163) [inline components/skeleton/style/index.ts:257]: 这里改错了。

Comment (li-jia-nan) [reply components/skeleton/style/index.ts:257]: 怪不得，我找了半天

Comment (li-jia-nan) [inline components/skeleton/style/index.ts:257]:
```suggestion
    [componentCls]: {
```

File: components/list/style/index.ts (hunk 1 above)
Comment (li-jia-nan) [reply components/list/style/index.ts:130]: 我去看看文档

Comment (li-jia-nan) [reply components/list/style/index.ts:130]: 没找到这个：

https://biomejs.dev/zh-cn/linter/rules

**Review by afc163 — APPROVED** [review]

**Pull request merged**